Assets/HoloToolkit-Examples/UX/Materials/3DTextSelawik.mat

```diff
@@ -0,0 +1,84 @@
+%YAML 1.1
```

Comment:
Duplicate of https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/UX/Materials/3DTextSelawik.mat

Reply:
@cre8ivepark I'll let you remove these duplicate materials since they may impact your test scenes and text prefabs.